source/features/git-checkout-pr.tsx

```diff
@@ -57,11 +57,11 @@ function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
 		</button>
 	);
 
-	const checkoutOptions = isLocalPr ? ['local'] : ['HTTPS', 'SSH'];
+	const checkoutOptions = isLocalPr() ? ['local'] : ['HTTPS', 'SSH'];
 	tabContainer.append(
 		<div hidden role="tabpanel" className="p-3">
 			<p className="text-gray text-small">
-				Run in your project repository{isLocalPr || ', pick either one'}
+				Run in your project repository{isLocalPr() || ', pick either one'}
 			</p>
 			{checkoutOptions.map(checkoutOption)}
 		</div>
```

Comment:
What do you think about splitting this like 

```js
 	tabContainer.append(
 		<div hidden role="tabpanel" className="p-3">
 			<p className="text-gray text-small">
 				Run in your project repository{isLocalPr() || ', pick either one'}
 			</p>
 			{isLocalPr() ? checkoutOption() : [checkoutOption('HTTPS'), checkoutOption('SSH')])}
 		</div>
	);
```

This will also let you:

- determine `isLocalPR` inside `checkoutOption` without actually calling `isLocalPR`, because `option` parameter will be missing.
- inline `isLocalPR` back into `handleMenuOpening` since now it would only used there.

Also keep "PR" uppercase, I think all acronyms should always be uppercase, or else we end up with `XMLHttpRequest`.

Reply:
I will give it a shot!

Reply:
> * because `option` parameter will be missing.

I added a local variable of `isLocalPR` since `!option` did not look right.